<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_CScreate_window</class>
 <widget class="QMainWindow" name="admin_CScreate_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(5, 130, 202);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="admincswdw_lbl_heading">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>581</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Creating Customer Account</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="admincswdw_btn_create">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 100, 148);

border:2px solid rgb(0, 150, 199);
border-radius:20px;
border-color:black;
}
QPushButton:hover {
     background-color: rgb(202, 240, 248);
     border:2px solid rgb(0, 150, 199);
}</string>
    </property>
    <property name="text">
     <string>CREATE CUSTOMER</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="admincswdw_linedit_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="admincswdw_btn_returnmain">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>500</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(0, 150, 199);
   border-color: rgb(66, 167, 255);
   border-bottom-color: rgb(255, 255, 255);
   border:2px solid rgb(202, 240, 248);
   border-radius:20px;
   border-color: black;
}

QPushButton:hover {
     background-color: rgb(202, 240, 248);
     border:2px solid rgb(202, 240, 248);
}
    
   </string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="admincswdw_btn_exit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>500</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(0, 150, 199);
   border-color: rgb(66, 167, 255);
   border-bottom-color: rgb(255, 255, 255);
   border:2px solid rgb(202, 240, 248);
   border-radius:20px;
   border-color: black;
}

QPushButton:hover {
     background-color: rgb(202, 240, 248);
     border:2px solid rgb(202, 240, 248);
}
    
   </string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="admincswdw_lbl_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="admincswdw_lbl_password">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="admincswdw_linedit_email">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="admincswdw_lbl_email">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="admincswdw_linedit_CSpassword_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="admincswdw_lbl_balance">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Current Balance:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="admincswdw_spinBox_balance">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="suffix">
     <string>â‚¬</string>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="admincswdw_lbl_result">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>588</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>admincswdw_btn_exit</sender>
   <signal>clicked()</signal>
   <receiver>admin_CScreate_window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>556</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
